pipelines/A5/queries.sql

```diff
@@ -5,3 +5,109 @@ JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
+  (T1.anzahl_renten - T2.anzahl_renten) AS differenz_anzahl_altersrenten_frauen_maenner
+FROM ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit AS T1
+JOIN ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit AS T2 ON T1.jahr = T2.jahr
+WHERE T1.geschlecht = 'Frau'
+  AND T2.geschlecht = 'Mann'
+  AND T1.rententyp = 'Altersrente' AND T2.rententyp = 'Altersrente'
+  AND T1.staatsangehorigkeit_kategorie = 'Staatsangehörigkeit - Total'
+  AND T2.staatsangehorigkeit_kategorie = 'Staatsangehörigkeit - Total'
+  AND T1.wohnsitzstaat_kategorie = 'Wohnsitzstaat - Total'
+  AND T2.wohnsitzstaat_kategorie = 'Wohnsitzstaat - Total';
+
+-- Gab es Rentenarten bei der der Anteil der an im Ausland lebenden ausländischer Staatsangehöriger im Jahr 2022 höher war als der der Anteil der im Inland lebenden Schweizer Staatsangeörigen: was waren die Zahlen?
+SELECT T1.rententyp,
+  SUM(T1.anzahl_renten) AS anzahl_renten_auslaender_im_ausland_2022,
+  SUM(T2.anzahl_renten) AS anzahl_renten_schweizer_im_inland_2022
+FROM ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit AS T1
+JOIN ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit AS T2 ON T1.rententyp = T2.rententyp
+WHERE T1.jahr = 2022 and T2.jahr = 2022
+  AND T1.staatsangehorigkeit_kategorie = 'Ausland' AND T1.wohnsitzstaat_kategorie = 'Ausland'
+  AND T2.staatsangehorigkeit_kategorie = 'Schweiz' AND T2.wohnsitzstaat_kategorie = 'Schweiz'
+  AND T2.geschlecht = 'Geschlecht - Total' AND T1.geschlecht = 'Geschlecht - Total'
+  AND T1.anzahl_renten > T2.anzahl_renten
+GROUP BY T1.rententyp;
+
+-- Wieviel Prozent der Rentenausgaben floss 2015 an Schweizer, die im Ausland leben und wieviel waren ?
+SELECT
+  ROUND((SUM(CASE WHEN jahr = 2015
+       AND staatsangehorigkeit_kategorie = 'Schweiz' AND wohnsitzstaat_kategorie = 'Ausland'
+       THEN rentensumme_in_tausend_franken ELSE 0 END) /
+   SUM(CASE WHEN jahr = 2015 AND staatsangehorigkeit_kategorie = 'Staatsangehörigkeit - Total'
+       AND wohnsitzstaat_kategorie = 'Wohnsitzstaat - Total'
+       THEN rentensumme_in_tausend_franken ELSE 0 END)) * 100, 2) AS prozent_rentenausgaben_fur_schweizer_im_ausland
+FROM ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit
+WHERE jahr = 2015
+  AND geschlecht = 'Geschlecht - Total';
+
+-- Wie hoch war der Frauenanteil bei den Zusatzrenten für Ehepartner?
+SELECT
+  ROUND(100 * SUM(CASE WHEN geschlecht = 'Frau'
+       THEN anzahl_renten ELSE 0 END) /
+   SUM(CASE WHEN geschlecht = 'Geschlecht - Total'
+       THEN anzahl_renten ELSE 0 END)
+  , 2) AS prozent_frauenanteil_bei_ehepartner_zusatzrenten
+FROM ahv_renten_nach_wohnsitz_und_staatsangehoerigkeit
+WHERE rententyp = 'Zusatzrente für Ehepartner'
+  AND staatsangehorigkeit_kategorie = 'Staatsangehörigkeit - Total' AND wohnsitzstaat_kategorie = 'Wohnsitzstaat - Total';
+
+-- In welchen drei Jahren gab es die höchsten Ausgaben für die Zusatzrenten für Waise und wie hoch waren diese?
```

Comment:
**suggestion:** specify chf as the unit, as technically keeping it in 1000CHF is also correct.
```suggestion
-- In welchen drei Jahren gab es die höchsten Ausgaben für die Zusatzrenten für Waise und wie hoch waren diese in CHF?
```